HelpMyStreetFE/HelpMyStreetFE/Controllers/AccountController.cs

```diff
@@ -164,6 +172,44 @@ public async Task<IActionResult> LoadAwardsComponent(CancellationToken cancellat
             return ViewComponent("Awards", new { userID = user.ID, cancellationToken = cancellationToken });
         }
 
+        [Route("print-job-details")]
+        [HttpGet]
+        public async Task<IActionResult> PrintJobDetails(string j, CancellationToken cancellationToken)
+        {
+            var jobID = Base64Utils.Base64DecodeToInt(j);
+            User user = await _authService.GetCurrentUser(HttpContext, cancellationToken);
+            if (user == null)
```

Comment:
This controller has the `[Authorize]` attribute, so you should find this check and redirect happen already, before the code in this method is executed.